model/src/main/kotlin/utils/FindingCurationMatcher.kt

```diff
@@ -19,12 +19,10 @@
 
 package org.ossreviewtoolkit.model.utils
 
-import java.nio.file.FileSystems
-import java.nio.file.Paths
-
 import org.ossreviewtoolkit.model.LicenseFinding
 import org.ossreviewtoolkit.model.config.LicenseFindingCuration
 import org.ossreviewtoolkit.model.licenses.LicenseFindingCurationResult
+import org.ossreviewtoolkit.utils.common.FileMatcher
```

Comment:
IMO the commit message should not refer to "Ant's path matcher", but just to our "FileMatcher".